knowledge_graph/routes.py

```diff
@@ -6,29 +6,31 @@
 
 from flask import request, make_response, abort, Response
 
+from typing import Tuple
+
 from knowledge_graph.Mind import Mind
 
-value_id_map = { 
-    1 : "conformity",
-    2 : "tradition",
-    3 : "benevolence",
-    4 : "universalism",
-    5 : "self-direction",
-    6 : "stimulation",
-    7 : "hedonism",
-    8 : "achievement",
-    9 : "power",
-    10 : "security"
+value_id_map = {
+    1: "conformity",
+    2: "tradition",
+    3: "benevolence",
+    4: "universalism",
+    5: "self-direction",
+    6: "stimulation",
+    7: "hedonism",
+    8: "achievement",
+    9: "power",
+    10: "security"
 }
 
 
 @app.route('/', methods=['GET'])
-def home():
-    return "<h1>API for climatemind ontology</h1>"
+def home() -> Tuple[str, int]:
+    return "<h1>API for climatemind ontology</h1>", 200
 
 
 @app.route('/ontology', methods=['GET'])
```

Comment:
it is usually plural, especially if the response is a list. /ontologies

Reply:
Sure, but there's only one ontology- I think 'ontologies' would be misleading.

Reply:
My thinking here was that you're querying (via the query string) the ontology